### PR TITLE
Proof the first half of the resources page

### PR DESCRIPTION
**Pull request by NathanLovato**
Removed quite a few passive voices, a few rare words, and shortened sentences, trying not to alter the meaning. @willnationsdev you can check it if you'd like. It's an important page and tut, but written for developers imo. I'm not sure the full page fits in the getting started section - a beginner programmer will have a hard time getting this one.

You mostly write in passive voice, and you tend to use words that many foreigners won't know (e.g. `mere`). Next time could you check your article in a readability checker or prose linter? E.g. using the hemingway test, langtool, proselint... not that you should write as they tell you, but they highlight passive voice, long sentences for the former...

### CODE REVIEW

**Comment by willnationsdev** [comment]
@NathanLovato XD I specifically pasted most of the non-code content into Hemingway so I could identify and fix problems like that. Guess some others slipped in after the initial post via the edits.

I had specifically wanted to put the content into a beginner section. I wrote it in the first place because new users frequently remarked on how the introductory docs never covered it. Consequently, they didn't know resource scripts were possible, nor how powerful they are.

I'll have to continue working on my technical writing ability. Thanks for the feedback.

File: getting_started/step_by_step/resources.rst [hunk 1]
```diff
@@ -6,106 +6,97 @@ Resources
...
-shows where it comes from. In this case, it comes from a png image.
+view and edit the resources properties.
 
 .. image:: img/resourcerobi.png
 
-When the resource comes from a file, it is considered an *external*
-resource. If the path property is erased (or it never had a path to
-begin with), it is considered a built-in resource.
+The path property tells us where the resource comes from. In this case it comes
+from a png image called ``robi.png``. When the resource comes from a file like
```
Comment (Calinou) [inline getting_started/step_by_step/resources.rst]: png image -> PNG image

File: getting_started/step_by_step/resources.rst [hunk 2]
```diff
@@ -6,106 +6,97 @@ Resources
...
-:ref:`Script <class_Script>`,
-:ref:`Mesh <class_Mesh>`,
-:ref:`Animation <class_Animation>`,
-:ref:`AudioStream <class_AudioStream>`,
-:ref:`Font <class_Font>`,
-:ref:`Translation <class_Translation>`,
-etc.
-
-When Godot saves or loads (from disk) a scene (.tscn or .scn), an image
-(png, jpg), a script (.gd) or pretty much anything, that file is
```
Comment (Calinou) [inline getting_started/step_by_step/resources.rst]: (png, jpg) -> (.png, .jpg)

Comment (NathanLovato) [reply getting_started/step_by_step/resources.rst]: This is a removed line, it's in red as you can see

**Comment by NathanLovato** [comment]
@willnationsdev At the moment you write like a computer scientist or an engineer, and it's normal: you have a solid technical understanding of the topic, I guess you are an engineer as well, right?

Writing accessible, beginner-friendly content takes a lot of practice and feedback from beginners themselves: neither of us devs remembers what it's like to learn to code for the first time. At some point you need to write a lot of content and get feedback from beginners to get there. Keep it up! Your content is always welcome, and you're improving already 😄

**Comment by NathanLovato** [comment]
@Calinou Thanks, fixed!

**Comment by NathanLovato** [comment]
@willnationsdev by the way, do you want me to proof the second half as well or should I leave it up to you? From line 200 and on.

**Pull request merged**